scripts/qiita

```diff
@@ -331,5 +349,18 @@ def start(port):
     click.echo("Qiita started on port %d" % port)
     tornado.ioloop.IOLoop.instance().start()
 
+
+# #############################################################################
+# WARE COMMANDS
+# #############################################################################
+
+@ware.command()
+@click.argument('sl_out_dir', required=True,
+                type=click.Path(resolve_path=True, readable=True, exists=True))
+def generate_demux(sl_out_dir):
+    """Generates the demux file for the split libraries output sl_out_dir"""
+    generate_demux_file(sl_out_dir)
+    click.echo("Demux file successfully generated")
```

Comment:
Maaaaayyybe worth to print the full file path of the demux file? Not a blocking comment and more of an idea.